tests/unit/test_admin_pipeline.py

```
@@ -0,0 +1,176 @@
+            print(f"Processing Time: {processing_time:.2f}s")
+            
+            if response.status_code == 200:
+                data = response.json()
+                
+                # Check pipeline indicators
+                response_data = data.get('data', {})
+                transcription_method = response_data.get('transcription_method', 'unknown')
+                admin_pipeline_used = response_data.get('admin_pipeline_used', False)
+                transcription_quality = response_data.get('transcription_quality', 'unknown')
+                
+                print(f"✅ Success!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)